invenio_accounts/cli.py

```diff
@@ -170,8 +171,14 @@ def users_deactivate(user):
 @with_appcontext
 def domains_create(domain):
     """Create domain."""
+    domain = domain.lower()
+
+    if DomainCategory.get(domain):
```

Comment:
if the model column has a unique attribute we don't need that check here, we can let the database handle the situation and catch the exception after line 184